Tour-Collective-Devs/tc_backend
# Rm nest user info

## Link to Ticket

Autolink format: hashtag and number, ex: #1094

Closes #26 

## Description of Proposed Changes
Added embeded user information on crew profile and employer profile serializers

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout rm-nest-user-info
```
1. load up browsable api
1. register a new user, navigate back to api root and make sure the email address for your new user is displayed in the upper right hand corner of the browsable api.
1. go and create an employer, the user field should be auto filled in with the user you are logged in as.
1. go through the same steps for crew member.

## Impacted Areas in Application
endpoints/serializers for crew member and employer

List general components of the application that this PR will affect:

## Mentions @username

Tag users that need to review this code

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
1. Every class must be documented with purpose, author, and methods.
1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
